src/components/Text/variants.ts

```diff
@@ -1,7 +1,8 @@
 import CSS from 'csstype';
-import { FOOTER_HEIGHT } from '../variables';
+import { FOOTER_HEIGHT, palette } from '../variables';
 
 export type TextVariant =
```

Comment:
Here same, create the TextVariant type from variants-object

Reply:
Done!